internal/pkg/cli/flag.go

```diff
@@ -142,6 +143,7 @@ Must be of the format '<keyName>:<dataType>'.`
 	memoryFlagDescription         = "Optional. The amount of memory to reserve in MiB for each task."
 	imageFlagDescription          = "Optional. The image to run instead of building a Dockerfile."
 	taskRoleFlagDescription       = "Optional. The role for the task to use."
+	executionRoleFlagDescription  = "Optional. The role that grants the Amazon ECS container agent permission to make AWS API calls on your behalf."
```

Comment:
nit: trying to stay ~80chars. I realize that it's taken from the CFN docs but it's also for the Fargate agent.
```suggestion
	executionRoleFlagDescription  = "Optional. The role that grants the container agent permission to make AWS API calls."
```